pkg/util/helper/workstatus.go

```diff
@@ -33,23 +43,27 @@ func AggregateResourceBindingWorkStatus(c client.Client, binding *workv1alpha2.R
 		klog.Errorf("Failed to get works by ResourceBinding(%s/%s): %v", binding.Namespace, binding.Name, err)
 		return err
 	}
-	targetWorks = append(targetWorks, workList.Items...)
```

Comment:
Why not remove `targetWorks` from `AggregateClusterResourceBindingWorkStatus` as well?